app/js/lib/foxbox/foxbox.js

```diff
@@ -389,6 +389,71 @@ export default class Foxbox extends Service {
     return this[p.db].setTag.apply(this[p.db], arguments);
   }
 
+  subscribeToNotifications(resubscribe = false) {
```

Comment:
Can you annotate this method, specifically to describe what is the `resubscribe` param for?